.swiftpm/xcode/xcshareddata/xcschemes/ClientRuntime-Package.xcscheme

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this shouldn't be required. XCode already supports Swift packages and for Linux is it of no use.

Reply:
I will remove this and the other `.xcscheme`s from this PR.  The auto-generated schemes will not run tests because the package definition includes multiple test targets and Xcode cannot auto-match them, hence the manually created schemes.

I think this can be worked around by creating non-shared schemes that live on the developer's machine only, so long as out `.gitignore` file is properly configured.

Reply:
Removed.